internal/cmd/connect/command_plugin_install.go

```diff
@@ -68,10 +63,10 @@ func (c *pluginCommand) newInstallCommand() *cobra.Command {
 		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireOnPremLogin},
 	}
 
-	cmd.Flags().String("component-dir", "", "The plugin installation directory. Defaults to $share/confluent-hub-components for archive deployment, and to /usr/share/confluent-hub-components for deb/rpm deployment.")
-	cmd.Flags().StringSlice("worker-configs", []string{}, "Comma-delineated list of paths to one or more Kafka Connect worker configuration files. Each worker file will be updated to load plugins from the component directory in addition to any preexisting directories.")
+	cmd.Flags().String("plugin-dir", "", "The plugin installation directory. Defaults to $share/confluent-hub-components for archive deployment, and to /usr/share/confluent-hub-components for deb/rpm deployment.")
```

Comment:
```suggestion
	cmd.Flags().String("plugin-dir", "", "The plugin installation directory. Defaults to $SHARE/confluent-hub-components for archive deployment, and to /usr/share/confluent-hub-components for DEB/RPM deployment.")
```

Reply:
Is there any way we can infer what type of deployment this is before the command is run, and set the default value depending on the deployment type?

Reply:
I just removed this part of the string altogether; I may add back a reworded version later if we want it.

Reply:
The comment still applies though, it might be nice to decide the default beforehand (this way it shows up for the user if they run `--help`)

Reply:
Although on second thought, I'm not sure how we'd choose which default to show in our autogenerated docs... (maybe there's a way we can hide it?)

Reply:
I'm inclined to just write a general message about how the command will select a default; I'll think about the wording.